plugins/scroll-options/src/ScrollBlockDragger.js

```diff
@@ -16,30 +16,80 @@ import * as Blockly from 'blockly/core';
  * someone is dragging it.
  */
 export class ScrollBlockDragger extends Blockly.BlockDragger {
+  /** @override */
+  constructor(block, workspace) {
+    super(block, workspace);
+
+    /**
+     * How much the block has been moved due to scrolling.
+     * @type {!Blockly.utils.Coordinate}
+     * @protected
+     * TODO: Do we still do underscore in Blockly samples?
```

Comment:
I do :D But according to the [styleguide](https://google.github.io/styleguide/jsguide.html#features-classes-fields) it only applies to private fields, not protected ones.